### ISSUE TITLE
Getting a cryptic error on boot 

### ISSUE THREAD

**Issue by rorykoehler** [keypup-io/cloudtasker#82]
First thanks for creating this fantastic gem. It's really well implemented and a joy to migrate to from Resque. 

I am experiencing a boot error when loading the cron schedule (`rescue in get_task`) for a resource project I don't recognize. 

```
{
insertId: "63c7f4110001ae265488ce27"
labels: {1}
logName: "projects/MY_PROJECT_ID/logs/run.googleapis.com%2Fstderr"
receiveTimestamp: "2023-01-18T13:28:49.450574676Z"
resource: {2}
textPayload: "/usr/local/bundle/gems/google-cloud-tasks-v2-0.6.0/lib/google/cloud/tasks/v2/cloud_tasks/client.rb:1537:in `rescue in get_task': 7:Permission denied on resource project 87ec33c5-3856-4826-a81e-483b47bbf038.. debug_error_string:{UNKNOWN:Error received from peer ipv4:142.250.159.95:443 {created_time:"2023-01-18T13:28:49.103259885+00:00", grpc_status:7, grpc_message:"Permission denied on resource project 87ec33c5-3856-4826-a81e-483b47bbf038."}} (Google::Cloud::PermissionDeniedError)"
timestamp: "2023-01-18T13:28:49.110118Z"
}
```

As far as I understand the `Permission denied on resource project 87ec33c5-3856-4826-a81e-483b47bbf038` should contain the my project ID? Instead there is a string which after analyzing everything looks to be the same format as a cloudtaskser job id however I cannot find a job with that ID either so perhaps that's just a coincidence as it also doesn't make sense in the context. ~~I do see 2 tasks appearing in the Cloud Task dashboard however they present the following error message when clicked on `Task does not exist. It can be dispatched or deleted.`~~ ... Ignore that I just deleted the Redis keys manually and tried redeploy and now there is nothing there so I am assuming it was grabbing the existing task schedule from the previous successful deployment.
 
I've tried numerous different service accounts and the problem persists. I have also tried rebuilding and deploying with the --no-cache option just in case something strange went on there but that also wasn't the issue.

Any pointers or ideas for further investigation in resolving this would be appreciated.

**Comment by alachaum** [comment]
Hey @rorykoehler, no worries. 

Let's check a few things:
1) Can you confirm your service is deployed on Cloud Run?
2) Did you properly specify the `gcp_project_id` config parameter in the initializer? Check the Cloudtasker [initializer section](https://github.com/keypup-io/cloudtasker#cloudtasker-initializer)
3) Are you able to pull the Cloud Tasks permissions of the service account you use for your service? This service account should have permissions to view/create tasks on the project you configured in the `cloudtasker.rb` initializer.

**Comment by rorykoehler** [comment]
Hi @alachaum thanks for your swift response.

1. Yes deployed to Cloud Run
2. I did specify the `gcp_project_id` as per your docs.
3. The service account has the Cloud Tasks Admin role. I have also tried assigning all the individual roles. 

I'm pretty sure this was working on the initial deploy with cloudtasker installed last week. Though I didn't do any extensive testing the deploy was successful.

**Comment by rorykoehler** [comment]
Hi @alachaum, 

I'm just following up to say that I have resolved my issue and leaving a comment in case other run into this issue. I suspect it had something to do with  the Alpine docker container and specifically the `gcompat` library as I was having issues with the build and installed that even though I was already using  `libc6-compat` which should theoretically cover my use case. Of course now the build suddenly works without `gcompat` again and when I pushed the build to Cloud Run everything works again.

Thanks again
